web-ui/src/main/resources/ui/lib/d3.tip.v0.6.3.js

```diff
@@ -0,0 +1,280 @@
+// d3.tip
+// Copyright (c) 2013 Justin Palmer
+//
+// Tooltips for d3.js SVG visualizations
+
+// Public - contructs a new tooltip
+//
+// Returns a tip
+d3.tip = function() {
+  var direction = d3_tip_direction,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'd3_tip_direction' is not in camel case.](https://www.codacy.com/app/frank_7/spawncamping-dds/file/2194327442/issues/source?bid=3046234&fileBranchId=3126677#l10)